src/lib/jsClient.ts

```diff
@@ -39,16 +41,17 @@ export const initFluenceClient = async (
     "off-aqua-logs": offAquaLogs,
   }: FluenceClientFlags & KeyPairFlag & OffAquaLogsFlag,
   maybeFluenceConfig: FluenceConfig | null
-): Promise<FluencePeer> => {
-  const client = createClient();
+): Promise<void> => {
   const relay = maybeRelay ?? getRandomRelayAddr(maybeFluenceConfig?.relays);
 
+  commandObj.log(`Fluence js-client connecting to: ${color.yellow(relay)}`);
```

Comment:
Maybe like that? Not sure what user should understand from `Fluence js-client`
```suggestion
  commandObj.log(`Connecting to: ${color.yellow(relay)}`);
```

Also, maybe let's mark log-level? Is there something like `commandObj.info`?

Reply:
Let's do it like you suggested. I wanna rework logging the same way it's done in js-client so it's consistent. Basically I wanna remove all such flags and show this logs when setting a topic in env variable